ci_test/install_installer_air_gapped.yaml

```
@@ -0,0 +1,15 @@
+---
+- hosts: all
+  roles:
+    - { role: "/root/project/" }
+  vars:
+    datadog_site: "ddog-gov.com"
+    datadog_api_key: "11111111111111111111111111111111"
+    datadog_yum_repo: "https://yum.datadoghq.com/stable/7/x86_64/"
+    datadog_yum_repo_config_enabled: true
+    datadog_enabled: false
+    datadog_skip_running_check: true
+    datadog_installer_registry: "install.datadoghq.com"
+    datadog_installer_version: 7.63
```

```suggestion
    datadog_installer_version: 7.62
```
Right now the default version is also 7.63, so setting it to an older version can help verify that its installing the `datadog_installer_version` rather than latest. with 7.62 here the installer version is `7.62.1-installer-0.11.0`